extensions/wikia/Email/Controller/CommentController.class.php

```diff
@@ -14,12 +14,25 @@
 	/** @var \Title */
 	protected $latestComment;
 
+	/** @var \Title */
+	protected $commentTitle;
+
 	public function initEmail() {
 		$titleText = $this->request->getVal( 'title' );
```

Comment:
Note that ```title``` here is the article being commented on, whereas ```currentRevId``` below is the latest revision ID for the comment that was left.